proto/cosmos/protocolpool/v1/tx.proto

```diff
@@ -60,6 +59,43 @@ message MsgCommunityPoolSpend {
 
 // MsgCommunityPoolSpendResponse defines the response to executing a
 // MsgCommunityPoolSpend message.
-//
-// Since: cosmos-sdk 0.50
-message MsgCommunityPoolSpendResponse {}
\ No newline at end of file
+message MsgCommunityPoolSpendResponse {}
+
+// MsgSubmitBudgetProposal defines budget proposal type.
+message MsgSubmitBudgetProposal {
+  option (cosmos.msg.v1.signer) = "authority";
+
+  // authority is the address that controls the module (defaults to x/gov unless overwritten).
+  string authority = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // recipient_address is the address of the recipient who can claim the budget.
+  string recipient_address = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // total_budget is the total amount allocated for the budget.
+  cosmos.base.v1beta1.Coin total_budget = 3;
+  // start_time is the time when the budget becomes claimable, represented in seconds since the Unix epoch.
+  // A value of `0` means the budget becomes claimable at the time of the first claim or governance proposal execution.
+  int64 start_time = 4;
```

Comment:
Why not using uint64 for all of those, actually, given that negative is always invalid.